Simplenote/AboutViewController.swift

```diff
@@ -28,7 +29,9 @@ class AboutViewController: NSViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
+        CrashLogging.crash()
```

Comment:
I'm assuming you'll remove this line before merging into `develop`? 😄

Reply:
Removed in 65bdace1adc4cd559b62b3de1e40b0a3b6cc7e72! 😅